KNOWN_ISSUES.md

```diff
@@ -50,11 +55,20 @@ for a full list of known issues.
       `Differentiable` conformance constraint.
       ([TF-208](https://bugs.swift.org/browse/TF-208))
 
+### TensorFlow Library
+
+* [ ] 🚧 **Model checkpointing and serialization APIs are missing.**
+      ([TF-388](https://bugs.swift.org/projects/TF/issues/TF-388))
```

Comment:
This JIRA link looks non-standard. I think below is the canonical version.
```suggestion
      ([TF-388](https://bugs.swift.org/browse/TF-388))
```

Reply:
Thanks! I changed all links to be reference-style in https://github.com/tensorflow/swift/pull/178/commits/b1bdaa003804894bdacec7d933c3986d0085045e (so that they don't take up so much space inline).

(It'd be nice to do the same for other Markdown files like `README.md`. I might do it for fun sometime.)